src/main/java/com/maplexpbar/MapleXPBarConfig.java

```diff
@@ -38,9 +38,17 @@ default Skill skill()
 	)
 	default boolean displayHealthAndPrayer() { return false; }
 
-	@Alpha
 	@ConfigItem(
 			position = 3,
+			keyName = "showPercentage",
+			name = "Show XP Percentage",
+			description = "Also shows XP percentage when hovering over bar"
+	)
+	default boolean showPercentage() { return true; }
```

Comment:
Notice it defaults to `true`, so it's an on by default / opt-out. I thought that might be best since OSRS's Show XP bars in the top-right corner feature a percentage on hover, too.